src/app/shared/services/switchboard-toastr.service.spec.ts

```diff
@@ -47,7 +47,7 @@ fdescribe('SwitchboardToastrService', () => {
   });
 
   it('should be run readAllItems', () => {
```

Comment:
Maybe something like below?
```suggestion
  it('should be run readAllItems', (done) => {
    service.success('test');
    const expectedResult = {message: 'test', type: 'toast-success', isNew: false};
    service.readAllItems();
    service.getMessageList().subscribe(list => {
      expect(list).toEqual([expectedResult]);
      done();
    });
  });
```